### PR TITLE
definitions for `ml:` concepts to be used by Croissant.

### PR DESCRIPTION
**Pull request by pierrot0**
So it seems to me that we don't intend to use them as JSON-LD `@type` but more as `dataType` values. So I kind of feel strange to have them in the spec itself, but could be convinced otherwise.

### CODE REVIEW

File: docs/definitions/validation_split.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# `ml:validation_split`
+
+Other known names: `dev`, `val`.
```
Comment (josvandervelde) [inline docs/definitions/validation_split.md:3]: Should `validation` be another known name?

File: docs/definitions/testing_split.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# `ml:testing_split`
+
+Other known names: `test`.
+
+The testing split is a data set used to provide an unbiased evaluation of a final model fit on the training data set. This data split might be hidden by the dataset author.
```
Comment (josvandervelde) [inline docs/definitions/testing_split.md:5]: nit: should maybe be "data set author" instead of "dataset author" for consistency?

**Review by josvandervelde — APPROVED** [review]
Great! Left some very minor comments

**Comment by pierrot0** [comment]
> Great! Left some very minor comments

Thanks!

**Pull request merged**